less/_components/quicksilver/dark-theme.less

```diff
@@ -0,0 +1,13 @@
+.prefers-color-scheme{
```

Comment:
@Linus-CS why is this wrapped in a class?

If you remove that class, we should be able to see your changes live in the preview: https://deploy-preview-154--eclipsefdn-solstice-assets.netlify.app

Reply:
@chrisguindon Oops I accidentally left that there. At the beginning I thought it maybe would be a good idea to wrap it in a class, so that instead of modifying the content directly, you would just use the class when creating a site to specify where changes should be applied after turning on dark theme. 

I will remove that and push the changes.